docs/modules/user-guide/partials/proc_developing-with-devfiles.adoc

```diff
@@ -0,0 +1,36 @@
+[id="proc_developing-with-devfiles_{context}"]
+= Developing with devfiles
+[role="_abstract"]
+
+Develop a node.js “Hello World” application, using the devfile specification. Developing this application introduces you to how a devfile automates and simplifies your development process.
+
+.Prerequisites
+
+* To develop an application by using the devfile specification, you need to be on a cluster. To create a cluster, install link:https://minikube.sigs.k8s.io/docs/start/[minikube start].
```

Comment:
```suggestion
* To develop an application by using the devfile specification, you need to create a Kubernetes cluster on your local machine. To create such a cluster, install link:https://minikube.sigs.k8s.io/docs/start/[minikube start].
```
- I'm not entirely clear what it means `to be on cluster`. Maybe it's better to specify that the user needs to use minikube to spin up a test or sandbox Kubernetes cluster on their local machine. My suggestion here is maybe too specific, but at least consider a more specific way to tell the user what they need to install.

Reply:
I think your rewrite makes senses and is better, thanks!